sys/shell/commands/sc_netif.c

```diff
@@ -267,6 +303,40 @@ static void _netif_set_addr(kernel_pid_t dev, ng_netconf_opt_t opt,
     puts("");
 }
 
+static void _netif_set_state(kernel_pid_t dev, char *state_str)
+{
+    ng_netconf_state_t state;
+    if ((strcmp("off", state_str) == 0) || (strcmp("OFF", state_str) == 0)) {
```

Comment:
seriously, we can not catch everything here - we are not programming Java after all! Every additional comparison takes away precious memory - and the netif shell commands already take ~3Kb on 32-bit Cortex platforms, so I say the user has just to live with these choices!